docs/docsite/rst/network/user_guide/platform_frr.rst

```diff
@@ -0,0 +1,72 @@
+.. _frr_platform_options:
+
+***************************************
+FRR Platform Options
+***************************************
+
+This page offers details on connection options to manage FRR using Ansible.
+
+.. contents:: Topics
+
+Connections Available
+================================================================================
+
+.. table::
+    :class: documentation-table
+
+    ====================  ==========================================
+    ..                    CLI
+    ====================  ==========================================
+    Protocol              SSH
+
+    Credentials           uses SSH keys / SSH-agent if present
+
+                          accepts ``-u myuser -k`` if using password
+
+    Indirect Access       via a bastion (jump host)
+
+    Connection Settings   ``ansible_connection: network_cli``
+
+    |enable_mode|         not supported
+
+    Returned Data Format  ``stdout[0].``
+    ====================  ==========================================
+
+.. |enable_mode| replace:: Enable Mode |br| (Privilege Escalation)
+
+
+Using CLI in Ansible
+====================
+
+Example CLI ``group_vars/frr.yml``
+----------------------------------
+
+.. code-block:: yaml
+
+   ansible_connection: network_cli
+   ansible_network_os: frr
+   ansible_user: frruser
+   ansible_password: !vault...
+   ansible_ssh_common_args: '-o ProxyCommand="ssh -W %h:%p -q bastion01"'
+
+- The `ansible_user` should be a part of the `frrvty` group and should have the default shell set to `/bin/vtysh`.
+- If you are using SSH keys (including an ssh-agent) you can remove the ``ansible_password`` configuration.
+- If you are accessing your host directly (not through a bastion/jump host) you can remove the ``ansible_ssh_common_args`` configuration.
+- If you are accessing your host through a bastion/jump host, you cannot include your SSH password in the ``ProxyCommand`` directive. To prevent secrets from leaking out (for example in ``ps`` output), SSH does not support providing passwords via environment variables.
+
+Example CLI Task
+----------------
+
+.. code-block:: yaml
+
+   - name: Gather FRR facts
+     frr_facts:
```

Comment:
should this use FQCN?

Reply:
Not sure about this. This guide would be valid for the FRR modules in ansible/ansible 2.9 as well and none of the other platform guides have the examples in fqcn format, afair.

Reply:
Hmm perhaps I misunderstood. I thought the purpose of column Q in John's internal spreadsheet (update platform guides) was specifically to have them use collections. 

Also, what you are changing here is the 2.10 FRR platform guide. You would need to backport this to 2.9 to have it applicable/available for 2.9.  

But this brings up a bigger issue. All of the 2.9 platform guides are... for 2.9 :-)  None of them have details about collections, yet  the 1.0.0 collections will all point to these guides. 

So for now, just keep it as you have it (no FQCN) and backport please.
I'll discuss more openly what we should do to cover the fact that the batch of 1.0.0 network collections will work with 2.9 and what if anything we should put in 2.9 platform docs.

Reply:
+1